common/src/test/scala/explore/undo/UndoerSpec.scala

```diff
@@ -9,40 +9,118 @@ import cats.effect.IO
 import cats.implicits._
 import cats.effect.concurrent.Ref
 import monocle.Iso
-import scala.concurrent._
-import ExecutionContext.Implicits.global
+import explore.components.undo.ListItem
 
 object UndoerSpec extends TestSuite {
 
   def id[A] = Iso.id[A].asLens.asGetter
 
+  def eqByRef[A]: A => A => Boolean = (a1: A) => (a2: A) => a1 == a2
```

Comment:
if it is by Ref, should it be `a1 eq a2`? What about an `eqByEq[A: Equal]`

Reply:
Yup, good catch, it's misnamed. And yes, `==` was just for tests, in real code we should use `Eq`, but I'll change the tests to that so that it sets a good example ;)